app/containers/Network/containers/AccessPoints/index.js

```diff
@@ -138,14 +156,20 @@ const AccessPoints = ({ checkedLocations }) => {
   }
 
   return (
-    <NetworkTable
-      tableColumns={accessPointsTableColumns}
-      tableData={equipData && equipData.filterEquipment && equipData.filterEquipment.items}
-      onLoadMore={handleLoadMore}
-      isLastPage={
-        equipData && equipData.filterEquipment && equipData.filterEquipment.context.lastPage
-      }
-    />
+    <>
+      <div className={styles.headerBtnContent}>
+        <ToggleButton activeTab="/network/access-points" />
+        <Button onClick={reloadTable} title="reload" icon={<ReloadOutlined />} />
+      </div>
+      <NetworkTable
+        tableColumns={accessPointsTableColumns}
+        tableData={equipData && equipData.filterEquipment && equipData.filterEquipment.items}
+        onLoadMore={handleLoadMore}
+        isLastPage={
+          equipData && equipData.filterEquipment && equipData.filterEquipment.context.lastPage
+        }
+      />
+    </>
```

Comment:
create a container in wlan-cloud-ui-library with this and pass props, both pages use the same code.